src/overlays/actors/ovl_Obj_Oshihiki/z_obj_oshihiki.h

```diff
@@ -6,11 +6,49 @@
 
 struct ObjOshihiki;
 
+typedef enum {
+    /* 0 */ PUSHBLOCK_SMALL_START_ON,
+    /* 1 */ PUSHBLOCK_MEDIUM_START_ON,
+    /* 2 */ PUSHBLOCK_LARGE_START_ON,
+    /* 3 */ PUSHBLOCK_HUGE_START_ON,
+    /* 4 */ PUSHBLOCK_SMALL_START_OFF,
+    /* 5 */ PUSHBLOCK_MEDIUM_START_OFF,
+    /* 6 */ PUSHBLOCK_LARGE_START_OFF,
+    /* 7 */ PUSHBLOCK_HUGE_START_OFF
+} PushBlockTypes;
+
+#define PUSHBLOCK_ON_SCENE       ( 1 << 0 )
```

Comment:
No need for spaces imo, tho we don't really have a choice anyway if we want to be consistent since clang-format will get rid of them (eg. when we have them in C files).
```suggestion
#define PUSHBLOCK_ON_SCENE       (1 << 0)
```